contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -495,6 +495,7 @@ func (s *SchedulerServer) prepareExecutorInfo(hks hyperkube.Interface) (*mesos.E
 	// running executors in a cluster.
 	execInfo.ExecutorId = executorinfo.NewID(execInfo)
 	execInfo.Data = data
+	log.V(1).Infof("started with executor id %v", execInfo.ExecutorId.GetValue())
```

Comment:
I was using V(2) for printing out error messages during problem scenarios, and V(1) for printing out more verbose messages during normal scenarios. I can make the change to either one if it is more conforming to the current coding style.